### ISSUE TITLE
Syntax Highlighting absent in jupyter lab even when jupyterlab-ihaskell installed?

### ISSUE THREAD

**Issue by Bodhijesus** [gibiansky/IHaskell#1202]
Hi,

I'm on a wsl installation of Ubuntu 18.04, and I made sure to follow all the steps on the page including installing the extension for jupyter lab, and I still get the following result.  To be sure nothing was wrong with the installation itself, I ran a normal jupyter notebook with the Haskell kernel and did get syntax highlighting.  I run jupyter lab simply be calling the `jupyter lab` command.

![image](https://user-images.githubusercontent.com/18375375/103556973-664f4080-4e67-11eb-8e53-f5cceb3e12fc.png)

**Comment by jamesdbrock** [comment]
In September, *IHaskell* was changed so that you no longer need the *jupyterlab-ihaskell* extension to get syntax highlighting. https://github.com/gibiansky/IHaskell/pull/1191

Instead call the `ihaskell` kernel with the option `--codemirror Haskell`.

You can add that option to the `/usr/local/share/jupyter/kernels/haskell/kernel.json` file, (or wherever your `kernel.json` file is stored, run the command `jupyter paths` to see your paths).

Here is an example of a `kernel.json` file for a `stack`-based IHaskell installation:

```
/usr/local/share/jupyter/kernels/haskell $ cat kernel.json 
{"display_name":"Haskell","argv":["/opt/stack/global-project/.stack-work/install/x86_64-linux-tinfo6/fffebb3fddcd52e7f38de3e253571498d665f4223658a7bafc9d5035b02e4ef3/8.8.4/bin/ihaskell","kernel","{connection_file}","--ghclib","/opt/stack/programs/x86_64-linux/ghc-tinfo6-8.8.4/lib/ghc-8.8.4","--codemirror","Haskell","+RTS","-M3g","-N2","-RTS","--stack"],"language":"haskell"}(base)
```

We should change the README instructions for IHaskell.

**Comment by Bodhijesus** [comment]
Thanks for the response!  Unfortunately, an awkward thing I found is that while jupyter lab certainly recognizes the haskell kernel, there is no folder for the kernel in `/usr/local/share/jupyter/kernels`, only python3.  I went ahead and uninstalled the _jupyterlab-ihaskell_ extension _and_ made its own folder inside and created a kernel.json file with those parameters you specified, but with no luck:

`{"display_name":"Haskell","argv":["/opt/stack/global-project/.stack-work/install/x86_64-linux-tinfo6/fffebb3fddcd52e7f38de3e253571498d665f4223658a7bafc9d5035b02e4ef3/8.8.4/bin/ihaskell","kernel","{connection_file}","--ghclib","/opt/stack/programs/x86_64-linux/ghc-tinfo6-8.8.4/lib/ghc-8.8.4","--codemirror","Haskell","+RTS","-M3g","-N2","-RTS","--stack"],"language":"haskell"}(base)`

**Comment by jamesdbrock** [comment]
> made its own folder inside and created a kernel.json file 

Yeah, copying the `kernel.json` file which I posted definitely won't work. You have to find out where Jupyter installed the `kernel.json` file for your IHaskell kernel.

Run the `jupyter paths` command and then look for `kernels/haskell/kernel.json` in the `data:` directories.

Then when you find your `kernels/haskell/kernel.json`, paste in the `,"--codemirror","Haskell"` argument to `ihaskell`.

**Comment by jamesdbrock** [comment]
To be honest installing and configuring and running Jupyter with extra kernels is a huge nuisance. For running IHaskell, I recommend this Nix derivation https://github.com/tweag/jupyterWith or this Docker image https://github.com/jamesdbrock/ihaskell-notebook .

**Comment by Bodhijesus** [comment]
Sorry for the long update to this, but I just wanted to update that this issue is fixed for me now.  I found that my kernel.json file was stored in .local/share/jupyter/kernels/haskell directory and added the needed argument.  Checked out some example code and it totally works now, thanks so much!

![image](https://user-images.githubusercontent.com/18375375/106426208-0f169e80-641a-11eb-9fa1-d55830002935.png)
